MapboxNavigation/NavigationViewController.swift

```diff
@@ -496,8 +496,8 @@ open class NavigationViewController: UIViewController {
                 let directions = routeController.directions
                 let route = routeController.routeProgress.route
                 let navigationViewController = NavigationViewController(dayStyle: DayStyle(demoStyle: ()), nightStyle: NightStyle(demoStyle: ()), directions: directions)
-                navigationViewController.startNavigation(with: route, routeController: routeController, locationManager: locationManager)
-                
+                navigationViewController.startNavigation(with: route, animated: true, routeController: routeController, locationManager: locationManager)
```

Comment:
Why do we pass animated: true in here instead of animated: animated?

Reply:
This is called in 

```swift
func carPlayManager(_ carPlayManager: CarPlayManager, didBeginNavigationWith routeController: RouteController, window: UIWindow) 
```

It's hidden in github if you don't show more. I think passing animated: true is the right choice here.